plugins/woocommerce/includes/class-wc-deprecated-filter-hooks.php

```diff
@@ -23,7 +23,6 @@ class WC_Deprecated_Filter_Hooks extends WC_Deprecated_Hooks {
 	protected $deprecated_hooks = array(
 		'woocommerce_account_orders_columns'              => 'woocommerce_my_account_my_orders_columns',
 		'woocommerce_structured_data_order'               => 'woocommerce_email_order_schema_markup',
-		'woocommerce_add_to_cart_fragments'               => 'add_to_cart_fragments',
```

Comment:
I'm curious why this and the below `add_to_cart_fragments` index needed to be removed here? Is it because this was a legacy way of deprecating filters and your current implementation using `wc_deprecated_hook` handles the deprecation of `add_to_cart_fragments`?

Reply:
This uses a hook itself, so it would not be possible to detect 3rd party use of `woocommerce_add_to_cart_fragments` with the `has_filter` function with this present. I moved the deprecation inline.